systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -1221,6 +1240,195 @@ void testResizeJbodVolumes() {
+                .endSpec()
+                .build());
+
+        // 1. check Kafka pods
+        List<Pod> kafkaPods = kubeClient(testStorage.getNamespaceName())
+                .listPodsByPrefixInName(testStorage.getBrokerComponentName());
+        // controller pods
+        kafkaPods.addAll(kubeClient(testStorage.getNamespaceName())
+                .listPodsByPrefixInName(testStorage.getControllerComponentName()));
+
+        for (Pod kafkaPod : kafkaPods) {
+            verifyPodAdditionalVolumes(testStorage.getNamespaceName(), kafkaPod, "kafka", secretMountPath, configMapMountPath,
+                    secretValueBase64, configMapValue, secretKeyBase64, configMapKey);
+        }
+
+        // 2. check Connect pods
+        List<Pod> connectPods = kubeClient(testStorage.getNamespaceName()).listPods(Labels.STRIMZI_NAME_LABEL, KafkaConnectResources.componentName(testStorage.getClusterName()));
+        for (Pod connectPod : connectPods) {
+            verifyPodAdditionalVolumes(testStorage.getNamespaceName(), connectPod, KafkaConnectResources.componentName(testStorage.getClusterName())
+                    , secretMountPath, configMapMountPath,
+                    secretValueBase64, configMapValue, secretKeyBase64, configMapKey);
+        }
+
+
+        // 3. check Bridge pods
+        List<Pod> bridgePods = kubeClient(testStorage.getNamespaceName()).listPods(Labels.STRIMZI_NAME_LABEL, KafkaBridgeResources.componentName(testStorage.getClusterName()));
+
+        for (Pod bridgePod : bridgePods) {
+            verifyPodAdditionalVolumes(testStorage.getNamespaceName(), bridgePod, KafkaBridgeResources.componentName(testStorage.getClusterName()), secretMountPath, configMapMountPath,
+                    secretValueBase64, configMapValue, secretKeyBase64, configMapKey);
+        }
+    }
+
+    /**
+     * Verifies the configuration and content of additional volumes mounted in a specified Kubernetes pod.
+     * This includes checks for both a secret and a config map to ensure they are mounted at the specified
+     * paths and contain the expected values.
+     *
+     * @param namespace             The Kubernetes namespace in which the pod is located.
+     * @param pod                   The pod object to verify.
+     * @param secretMountPath       The mount path for the secret volume.
+     * @param configMapMountPath    The mount path for the config map volume.
+     * @param secretValueBase64     The expected Base64-encoded value of the secret to be verified.
+     * @param configMapValue        The expected value of the config map to be verified.
+     * @param secretKeyBase64       The key within the secret volume used to access the secret content.
+     * @param configMapKey          The key within the config map volume used to access the config map content.
+     */
+    private void verifyPodAdditionalVolumes(final String namespace, final Pod pod, final String containerName, final String secretMountPath,
```

Comment:
Because you are providing a lot of parameters to this method and the method is used exactly once, would it make sense to have it as part of the test? Or maybe split it to check for Secret and for ConfigMap?